src/entities/repos/submission_repo.py

```diff
@@ -140,6 +147,39 @@ async def update_task_state(
     await upsert_helper(session, task, FilingTaskProgressDAO)
 
 
+async def update_contact_info(
+    session: AsyncSession, lei: str, filing_period: str, new_contact_info: ContactInfoDTO
+) -> ContactInfoDAO:
+    filing = await get_filing(session, lei=lei, filing_period=filing_period)
+    found_contact_info = await query_helper(session, ContactInfoDAO, filing=filing.id)
```

Comment:
the filing object should contain the contact_info already, right? so do we need to query again directly on the ContactInfoDAO?